Islandora-Devops/isle-dc
# Fixing initial install options

This MR addresses the issues raised in #244 and other places regarding upstream changes breaking the `make local_install_profile_demo` command.

To test this MR, please check:

`make local`
`make demo`
`make local_install_profile_demo`

For bonus points I guess also `make demo_install_profile_demo` but I sort of think we should pull that out, I don't really see the use for it, honestly.

The `install_profile` commands will fail until this is accepted - it's the root cause of most of the issues. Islandora_defaults changed a storage type, and we're just now catching up to it, I guess...
https://github.com/Islandora-Devops/islandora_install_profile_demo/pull/16
